build_cpp.sh

```diff
@@ -91,10 +90,6 @@ echo
 echo -e '\e[34mRunning catkin build.\e[39m'
 echo
 
-# Use ROS_PARALLEL_JOBS=-j1 to compile with only one core (Useful to point errors in catkin_make)
-export ROS_PARALLEL_JOBS="-j$PARALLEL_JOBS -l$PARALLEL_JOBS"
-# export ROS_PARALLEL_JOBS="-j1"
-
```

Comment:
Like this `ROS_PARALLEL_JOBS` as set in [do_everything.sh](https://github.com/Intermodalics/ros_android/blob/df883c7d8cd0f8d04094c22fc1ca803b191d10d4/do_everything.sh#L9) is not used at all for the catkin workspace. The default behavior of `catkin_tools` is the same, but it is possible to pass `$ROS_PARALLEL_JOBS` to `catkin` as a command line argument (before `--cmake-args`) for consistency with other build steps. It forwards the `-j` and `-l` arguments to `make`:

```sh
catkin build -j$PARALLEL_JOBS -l$PARALLEL_JOBS
```

Reply:
Alright, let's use the same variable for the `catkin` workspace as well for consistency.